activerecord/test/cases/store_test.rb

```diff
@@ -8,7 +8,12 @@ class StoreTest < ActiveRecord::TestCase
   fixtures :'admin/users'
 
   setup do
-    @john = Admin::User.create!(name: "John Doe", color: "black", remember_login: true, height: "tall", is_a_good_guy: true)
+    @john = Admin::User.create!(
+      name: "John Doe", color: "black", remember_login: true,
+      height: "tall", is_a_good_guy: true,
+      mother_name: "Mary", darling_name: "Lily",
+      darling_birthday: "1997-11-1"
```

Comment:
Update the attribute names to be gender neutral as in the documentation comment above and the rest of the tests as well.